source/Halibut/ServiceModel/ServiceInvoker.cs

```diff
@@ -13,10 +13,12 @@ namespace Halibut.ServiceModel
     public class ServiceInvoker : IServiceInvoker
     {
         readonly IServiceFactory factory;
+        readonly Guid halibutRuntimeProcessIdentifier;
```

Comment:
nit: This could be renamed to "HalibutRuntimeInstanceId" as Process does sound like a OS process. We can have many HalibutRuntime instances under a single CLR and each of those have their own Id